planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -578,23 +578,32 @@ class AvoidanceModule : public SceneModuleInterface
 
   /**
    * @brief reset registered shift lines.
-   * @param path shifter.
+   * @details reset only when the base offset is zero. Otherwise, sudden steering will be caused;
    */
-  void removeAllRegisteredShiftPoints(PathShifter & path_shifter)
+  void removeRegisteredShiftLines()
   {
+    constexpr double THRESHOLD = 0.1;
+    if (std::abs(path_shifter_.getBaseOffset()) > THRESHOLD) {
```

Comment:
Is the reason you set the threshold because of the assumption that egos would be able to adapt to adjacent shift lines if the offset is less than 0.1 m?

Reply:
I disccussed ota-san and it was true in that we don't have to be worred about the sudden steering.

Reply:
When the module removes registered shif lines, it outputs the path whose offset is `path_shifter_.getBaseOffset()`.
So, I added a guard in order to prevent huge path shifting.